scss/main.scss

```diff
@@ -102,3 +102,5 @@ nav {
     }
   }
 }
+
+@import 'responsive';
```

Comment:
So if this line is moved to top, it won't override the styles written in this file.